dockerfiles/Dockerfile.openvino

```diff
@@ -1,92 +1,50 @@
 #-------------------------------------------------------------------------
-# Copyright(C) 2019 Intel Corporation.
-# Licensed under the MIT License.
+# Copyright(C) 2021 Intel Corporation.
+# SPDX-License-Identifier: MIT
 #--------------------------------------------------------------------------
 
-FROM ubuntu:18.04
+ARG OPENVINO_VERSION=2021.3
 
-ARG DEVICE=CPU_FP32
-ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime.git
-ARG ONNXRUNTIME_BRANCH=master
+# Build stage
+FROM openvino/ubuntu18_runtime:${OPENVINO_VERSION} AS builder
 
-WORKDIR /code
-ARG MY_ROOT=/code
+ENV WORKDIR_PATH=/home/openvino
+WORKDIR $WORKDIR_PATH
+ENV DEBIAN_FRONTEND noninteractive
 
-ENV PATH /opt/miniconda/bin:/code/cmake-3.20.3-linux-x86_64/bin:$PATH
-ENV LD_LIBRARY_PATH=/opt/miniconda/lib:/usr/lib:/usr/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+ARG DEVICE=CPU_FP32 
+ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime.git
+ARG ONNXRUNTIME_BRANCH=master
 
-ENV INTEL_OPENVINO_DIR=/opt/intel/openvino_2021.3.394
 ENV InferenceEngine_DIR=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/share
-ENV IE_PLUGINS_PATH=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/lib/intel64
-ENV LD_LIBRARY_PATH=/opt/intel/opencl:${INTEL_OPENVINO_DIR}/inference_engine/external/gna/lib:${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/mkltiny_lnx/lib:$INTEL_OPENVINO_DIR/deployment_tools/ngraph/lib:${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/omp/lib:${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/tbb/lib:${IE_PLUGINS_PATH}:${LD_LIBRARY_PATH}
-ENV OpenCV_DIR=${INTEL_OPENVINO_DIR}/opencv/share/OpenCV
-ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/opencv/lib:${INTEL_OPENVINO_DIR}/opencv/share/OpenCV/3rdparty/lib:${LD_LIBRARY_PATH}
-ENV HDDL_INSTALL_DIR=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/hddl
-ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/hddl/lib:$LD_LIBRARY_PATH
 ENV ngraph_DIR=${INTEL_OPENVINO_DIR}/deployment_tools/ngraph/cmake
-ENV LANG en_US.UTF-8
-ENV DEBIAN_FRONTEND=noninteractive
 
+USER root
+RUN apt update; apt install -y git protobuf-compiler libprotobuf-dev
+RUN git clone --recursive -b ${ONNXRUNTIME_BRANCH} ${ONNXRUNTIME_REPO} 
+RUN /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh
+RUN ln -s cmake-* cmake-dir
+RUN python3 -m pip install wheel
+ENV PATH=${WORKDIR_PATH}/cmake-dir/bin:$PATH
+RUN cd onnxruntime/cmake/external/onnx && python3 setup.py install 
+RUN cd onnxruntime && ./build.sh --config Release --update --build --parallel --use_openvino ${DEVICE} --build_shared_lib --build_wheel 
```

Comment:
Should we go for RelWithDebInfo Release as default ?

Reply:
We cannot include Debug symbols according to SDL requirement. So we only have to build in Release config.